website/docs/mlflow/autologging.md

```diff
@@ -0,0 +1,77 @@
+---
+title: SynapseML Autologging
+description: SynapseML autologging
+---
+
+## Automatic Logging
+
+[MLflow automatic logging](https://www.mlflow.org/docs/latest/tracking.html#automatic-logging) allows you to log metrics, parameters, and models without the need for explicit log statements.
+For SynapseML we support autologging for all of our existing models.
+
+To enable autologging for SynapseML:
+1. Download this customized [log_model_allowlist file](https://mmlspark.blob.core.windows.net/publicwasb/log_model_allowlist.txt) and put it at a place that your code have access to.
+2. Set spark configuration `spark.mlflow.pysparkml.autolog.logModelAllowlistFile` to the path of your `log_model_allowlist.txt` file. 
+3. Call `mlflow.pyspark.ml.autolog()` before your training code to enable autologging for all supported models.
+
+Note:
+1. If you want to support autologging of pyspark models that's not in the log_model_allowlist file, you can modify the file to add them.
+2. If you've enabled autologging, then please don't write explicit `with mlflow.start_run()` as the behavior would be strange.
+
+
+## Configuration process in Databricks as an example
+
+1. Install MLflow via `%pip install mlflow`
+2. Upload your customized `log_model_allowlist.txt` file to dbfs by clicking File/Upload Data button on Databricks UI.
+3. Set Spark configuration:
+```
+spark.conf.set("spark.mlflow.pysparkml.autolog.logModelAllowlistFile", "/dbfs/FileStore/PATH_TO_YOUR_log_model_allowlist.txt")
```

Comment:
does this accept URLs? Perhaps we can host a reasonable default in our blob!

Reply:
We might also want to mention this can be set in cluster configs too

Reply:
Thanks for this idea! I'll raise a PR to mlflow to make the URL work lol this sounds so reasonable. And I just tested that the above spark.conf.set doesn't work as the cluster is already started, I'll change it to add spark config inside cluster configuration.

Reply:
https://github.com/mlflow/mlflow/pull/5857